src/index.ts

```diff
@@ -209,4 +209,26 @@ app.get("/attendances/:course_id", async (c) => {
   });
 });
 
+app.get("/me", async (c) => {
+  const HonoSession = c.get("session");
+  const login = HonoSession.get("login") as Session | null;
+  if ( !login ){
+    return c.text("Unauthorized", 401);
+  }
+  const entry = await c.env.DB
+  .prepare("SELECT name FROM account WHERE id = ?")
+  .bind(login.account.id)
+  .first();
+
+  if ( entry === null ){
+    throw new Error("Entry was invalid");
+  }
+
+  const name = entry;
```

Comment:
```suggestion
  const { name } = entry;
```
`entry` は SELECT したすべての列なので, 分割代入などで取り出す必要があると思います

Reply:
nameだけSELECTした場合にも分割代入が必要なのですか？

Reply:
必要です. [`first` メソッドのドキュメント](https://developers.cloudflare.com/d1/platform/client-api/#await-stmtfirstcolumn) によると, `first` に引数を渡さない場合は全てのカラムをキーバリューに格納したオブジェクトを返し, 引数を渡すとその名前のカラムの値だけを返すようです.

Reply:
nameのみSELECTした場合でも分割代入が必要ですか？

Reply:
はい, 必要です. [`first` メソッドのドキュメント](https://developers.cloudflare.com/d1/platform/client-api/#await-stmtfirstcolumn) によると, `first` に引数を渡さない場合は全てのカラムをキーバリューに格納したオブジェクトを返し, 引数を渡すとその名前のカラムの値だけを返します.

Reply:
ありがとうございます。